turbinia/turbiniactl.py

```diff
@@ -658,7 +664,7 @@ def process_args(args):
       process_evidence(
           args=args, source_path=source_path, name=name, format=format,
           group_id=group_id, client=client, filter_patterns=filter_patterns,
-          yara_rules=yara_rules, browser_type=browser_type)
+          yara_rules=yara_rules, browser_type=browser_type, group_name=args.group_name, reason=args.reason, all_args=sys.argv)
```

Comment:
As mentioned above we can probably do something like `' '.join(sys.argv)` when passing this in.